libpod/container_api.go

```diff
@@ -330,9 +330,10 @@ func (c *Container) Exec(tty, privileged bool, env, cmd []string, user string) e
 	}
 
 	pidFile := c.execPidPath(sessionID)
-	// 1 second seems a reasonable time to wait
-	// See https://github.com/containers/libpod/issues/1495
-	const pidWaitTimeout = 1000
+	// 3 second seems a reasonable time to wait
+	// https://github.com/containers/libpod/issues/1495
+	// https://github.com/containers/libpod/issues/1816
+	const pidWaitTimeout = 3000
```

Comment:
Is this overrideable via the command line or is it in the conf file?  If not, should it be?

Reply:
Although it would have helped in this issue, I don't want users to worry about such a technical detail. I'd prefer to set the limit much higher to be more sure of a general hiccup.

Reply:
I don't think it is override-able via the CLI and I'm not sure a user would need to do it, to me it looks like an internal parameter.

Reply:
Agree with not overriding by CLI - this is tucked too far inside, too deep of an internal configuration detail

Reply:
I'm fine increasing it to 60000ms. Are we all good with it?

Reply:
How about we add the limit to libpod.conf?

Reply:
yes please, I agree with increasing it.  Better waiting longer than killing a container too early.

Reply:
I pushed a new patchset with 60s and I didn't make it override-able via CLI nor libpod.conf. If you want to do it, let me know.